src/main/prometheus/operator.ts

```diff
@@ -50,7 +50,7 @@ export class PrometheusOperator extends PrometheusProvider {
           case "memoryAllocatableCapacity":
             return `sum(kube_node_status_allocatable{node=~"${opts.nodes}", resource="memory"})`;
           case "cpuUsage":
-            return `sum(rate(node_cpu_seconds_total{node=~"${opts.nodes}", mode=~"user|system"}[${this.rateAccuracy}]))`;
+            return `sum(rate(node_cpu_seconds_total{mode=~"user|system"}[${this.rateAccuracy}])* on (pod,namespace) group_left(node) kube_pod_info{node=~"${opts.nodes}"})`;
```

Comment:
The queries are now wrong again 😉 
It should be reversed, join on `node`, take the specific labels
```
on (node) group_left(pod, namespace)
```

Reply:
It does not work for us. Many reports about the queries you provided as failing that is why we reverted them.

Reply:
I agree that they need to be reintroduced to fix the compatibilty, but they need to get applied correctly to get it to work.

Reply:
ahhh, so `node_cpu_seconds_total` should contain `pod` and `namespace`. Maybe a comment about that would be useful. The node exporter in my setup doesn't have those labels. That also belongs into the documentation.

Reply:
Ah thanks for the info, would you mind making an issue to add that to the documentation?

Reply:
@marcbachmann how did you install prometheus operator?

Reply:
I have an external node exporter running natively on the hosts.
Therefore it only has the `node` label. I'm doing this because I have still some other services like databases outside kubernetes.

Might be an edge case and I could migrate that into the cluster.